app/queries/drug_stocks_query.rb

```diff
@@ -64,16 +78,16 @@ def drug_consumption_report
   end
 
   memoize def repository
-    Reports::Repository.new(Region.where(source: @facilities), periods: Period.month(@for_end_of_month))
+    Reports::Repository.new(Region.where(source: @facilities) + [@district], periods: Period.month(@for_end_of_month))
```

Comment:
Can we extract this region math to a semantically named method?

Reply:
Additionally, would accepting a list of facility _regions_ make this less awkward?

Reply:
We're trying to make a single call to repository to extract the patients under care number for both facilities and districts. The regions don't have anything in common, it's just a list of regions we want to lookup in the repository. A method like `repository_regions` doesn't seem very useful.

We can also make two repository instances: `facilities_repository` and a `district_repository`, although that doesn't buy us much semantically.

wdyt of 94df952?

Reply:
Oh this is only for patients under care? Interesting. Yeah that commit looks good 😅